lib/src/box.dart

```diff
@@ -186,6 +186,28 @@ class Box<T> {
     }
   }
 
+  bool contains(int id) {
+    Pointer<Uint8> _contains = Pointer<Uint8>.allocate();
```

Comment:
I've forgotten to bring this up yesterday. Could you please not use an underscore for local variables? It may be confused with private instance vars, especially in larger methods

Reply:
Yeah sorry, "fixed" that :+1: